### DISCUSSION TITLE
Genetic Material is way too grindy, frustrating, and in most cases, weak.

### DISCUSSION THREAD

**Discussion by Rotiart-Drol** [FakeFishGames/Barotrauma#12713] [Ideas and suggestions]
Genetic Material has existed for a long time now, for all intents and purposes as a mechanic for a specific tree for Medics, despite ostensibly being a part of the base game. Even as a talent-adjacent mechanic, in a scenario where a player has all the relevant talents, and all the relevant buffs, the value you get out of farming Genetic Material, as opposed to going down the Chemist tree for Endocrine boosters, is comparable at best, but unfortunately in most cases, it will simply turn out to be a waste of time

To compare:

An Endocrine Booster, for the relatively low sum of 2 Sulphurite Shards, 2 Paralyxis and a shot of Deusizine, most of which is easy enough to farm for, can give the target a _permanent_ talent from a pool containing such giants like Lone Wolf, Mudraptor Wrestler, Example of Health, and Swole. Potentially you can get all of those talents at the same time, all of which perform more impressively than most of the Genetic Material you are going to be able to get your hands on.

Genetic Material, after you plan _days_ ahead and spend _days_ grinding, praying RNGesus doesn't say "Screw you for trying" might net you a very good 25% damage resistance from Moloch Genes, a good amount of regeneration from Hammerhead Matriarch Genes, or the ability for Melee to be able to compete with Firearms from Hammerhead or Hunter Genes, all of which are lost on death, can be permanently stunted by pure luck without a _meaningful_ amount of player agency, with the addition of a debuff being now attached to the undesirable gene

I tested just how bad the Genetic Material grind is, and with not only an impossible amount of cross talents on a single character, with every relevant buff, and by spawning in a steady stream of unidentified genetic material, it still took me nearly an hour of just refining before I even got a single gene to max, a gene that, statistically in a campaign, would be very undesirable to begin with. There is also something to be said for the fact that talents like Gene Harvester and Gene Splicer barely feel like a drop in the bucket in the face of the 40 or so genetic material you'll end up dozing through each time you want to max out a gene that you want. I can't even imagine the heartbreak that would ensue when someone has grinded for a desirable gene for days, only for that gene to completely randomly taint at the final hurdle. That is not a feature that makes the game better, nor is that a feature that belongs in a coop game with lobbies this large. Most players, even a good chunk of dedicated players, are never going to experience a gene's impact on their game in a way they can notice, and even if they do, it could be ripped away permanently if they die for whatever reason

If Genetic Material is this grindy in order to gatekeep something game breakingly powerful, a player should not even have the option of obtaining that thing in the first place. That being said, I don't think any genes, aside from Moloch Genes, are inherently unhealthy, and certainly for most genes, powerful is not a word that is suited to describe them even were they maxxed

**Comment by Hazmat616** [comment]
Don't forget zealot robes or husk symbiosis can further punish you for trying to use genetics as they can simply circumvent it's need. For these reasons I feel zealot robes should take up the gene splicer slot as well as have a permanent taint

**Comment by gerakis** [comment]
The "level 1" genes are the only good ones, like husk or regenerating health gene. You can eventually regenerate even from a level 1 gene. Considering that you can just patch up for a bit of money from station doctors even that one is pretty worthless.
